presto-oracle/src/test/java/io/prestosql/plugin/oracle/BaseOracleIntegrationSmokeTest.java

```diff
@@ -90,9 +99,44 @@ public void testShowCreateTable()
                         ")");
     }
 
+    @Test
+    public void testDropTable()
+    {
+        String tableName = "test_drop" + randomTableSuffix();
+        assertUpdate(format("CREATE TABLE %s AS SELECT 1 test_drop", tableName), 1);
+        assertTrue(getQueryRunner().tableExists(getSession(), tableName));
+
+        assertUpdate("DROP TABLE " + tableName);
+        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
+    }
+
+    @Test
+    public void testInsert()
+    {
+        try (TestTable table = new TestTable(onOracle(), getUser() + ".test_insert", "(x number(19), y varchar(100))")) {
```

Comment:
`io.prestosql.testing.AbstractTestDistributedQueries#testInsert` is run for Oracle as well